doc/sphinx/user/authors.md

```diff
@@ -0,0 +1,15 @@
+(sec:authors)=
+# ASPECT community
+
+## Principal developers
+
+* Wolfgang Bangerth
+* Juliane Dannberg
+* Menno Fraters
+* Rene Gassmöller
+* Anne Glerum
+* Timo Heister
+* Bob Myhill 
+* John Naliboff
+
+**with contributions by:** Jacqueline Austermann, Magali Billen, Markus B&uuml;rg, Thomas Clevenger, Samuel Cox, William Durkin, Grant Euen, Thomas Geenen, Ryan Grove, Eric Heien, Ludovic Jeanniot, Louise Kellogg, Scott King, Martin Kronbichler, Marine Lasbleis, Haoyuan Li, Shangxin Liu, Hannah Mark, Elvira Mulyukova, Bart Niday, Jonathan Perry-Houts, Elbridge Gerry Puckett, Tahiry Rajaonarison, Fred Richards, Jonathan Robey, Ian Rose, Max Rudolph, Stephanie Sparks, D. Sarah Stamps, Cedric Thieulot, Wanying Wang, Iris van Zelst, Siqi Zhang'
```

Comment:
This is the list of authors of the documentation, maybe we need to make this more obvious by adding a sentence on this page. Why are you not listed? :-)

Reply:
Not sure how we get this list. I only copied the list from `conf.py`.